### DISCUSSION TITLE
Why have sourceDirectories the maven values and not the tycho ones?

### DISCUSSION THREAD

**Discussion by pjljvandelaar** [eclipse-tycho/tycho#3048] [Q&A]
When looking at the value of 
```
   <sourceDirectory>...\src\main\java</sourceDirectory>
    <testSourceDirectory>...\src\test\java</testSourceDirectory>
```
I am surprised that they have the maven defaults and not tycho ones:
```
   <sourceDirectory>...\src</sourceDirectory>
    <testSourceDirectory>...\src-test</testSourceDirectory>
```

In the latter case, I could also use the variable `${project.build.sourceDirectory}`.
See e.g. https://stackoverflow.com/questions/2362652/excluding-classes-in-maven-checkstyle-plugin-reports for relevant use case.

**Comment by laeubi** [comment]
Please note that not Tycho is making any defaults, the are inherited form JDT/PDE ...

Beside that it all depends on your configuration, so the best would be to share a minimal example, also not that Eclipse is able to define multiple source directories while maven pom does not directly expose that.
